### PR TITLE
doc: app_dev: bootloaders: sysbuild: Fix variable usage

### PR DESCRIPTION
**Pull request by nordicjm**
Fixes an issue with a variable being used

NCSIDB-1326

### CODE REVIEW

**Review by hellesvik-nordic — APPROVED** [review]

File: doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst [hunk 1]
```diff
@@ -263,7 +263,7 @@ You can set the option in :file:`sysbuild.conf` or using the command line:
 
       Escaped quotations avoid malformed-string warnings from Kconfig.
 
-The path of the key must be an absolute path, though ``${APPLICATION_CONFIG_DIR}`` can be used to get the path of the application configuration directory to use keys relative to this directory.
+The path of the key must be an absolute path, though the escaped ``${APPLICATION_CONFIG_DIR}`` CMake variable (by replacing the ``$`` with ``\$``) can be used to get the path of the application configuration directory to use keys relative to this directory.
```
Comment (annwoj) [inline doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst:266]:
```suggestion
The path of the key must be an absolute path. 
To utilize keys relative to the application configuration directory, use the escaped CMake variable ``\${APPLICATION_CONFIG_DIR}`` in the path by replacing ``$`` with ``\$``). 
```
tried to make it flow better, please double-check the meaning

**Comment by greg-fer** [comment]
Anna is already reviewing, so I'm removing myself.

File: doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst (hunk 1 above)
Comment (nordicjm) [reply doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst]: Fixed

File: doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst [hunk 2]
```diff
@@ -263,7 +263,8 @@ You can set the option in :file:`sysbuild.conf` or using the command line:
 
       Escaped quotations avoid malformed-string warnings from Kconfig.
 
-The path of the key must be an absolute path, though ``${APPLICATION_CONFIG_DIR}`` can be used to get the path of the application configuration directory to use keys relative to this directory.
+The path of the key must be an absolute path.
+The To utilize keys relative to the application configuration directory, use the escaped CMake variable ``\${APPLICATION_CONFIG_DIR}`` in the path by replacing ``$`` with ``\$``).
```
Comment (annwoj) [inline doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst:267]:
```suggestion
To utilize keys relative to the application configuration directory, use the escaped CMake variable ``\${APPLICATION_CONFIG_DIR}`` in the path by replacing ``$`` with ``\$``.
```

Comment (nordicjm) [reply doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst]: Silly mistake that, fixed

Comment (annwoj) [reply doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst]: the leftover bracket wasn't removed - can we get rid of that?

Comment (nordicjm) [reply doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst]: Now should be fixed

**Pull request merged**